laos_gggi/world_bank_data_loader.py

```diff
@@ -0,0 +1,60 @@
+import os
+from os.path import exists
+from pandas_datareader import wb
+import pandas as pd
+from const_vars import COUNTRIES_ISO, ISO_DICTIONARY
+
+
+def download_wb_data(output_path="../data"):
+    path_to_wb_data = output_path + "/world_bank.csv"
+    if not exists(output_path):
+        os.makedirs(output_path)
+        # Importing data
+        wb_df = wb.download(
+            indicator=["EN.POP.DNST", "NY.GDP.PCAP.KD"],
```

Comment:
add a global variable `WB_INDICATORS = ["EN.POP.DNST", "NY.GDP.PCAP.KD"]` and use it here

Reply:
Done!